cli/cmd/plugin/ui/web/tanzu-ui/src/shared/constants/defaults/azure.defaults.tsx

```diff
@@ -14,16 +17,16 @@ export const AZURE_DEFAULT_VALUES = {
     WORKER_NODE_SUBNET: 'tanzu-worker-node-subnet-default',
     WORKER_NODE_SUBNET_CIDR: '10.0.1.0/24',
     // Network
-    CNI_TYPE: 'antrea',
+    CNI_TYPE: 'antrea', // TODO: refactor to use CniProviders Const
     CLUSTER_POD_CIDR: '100.96.0.0/11',
     CLUSTER_SERVICE_CIDR: '100.64.0.0/13',
     ACTIVATE_PROXY_SETTINGS: false,
 };
 
 const AZURE_DEFAULT_INSTANCE_TYPES = new Map<string, string>([
-    ['SINGLE_NODE', 'Standard_D2s_v3'],
-    ['HIGH_AVAILABILITY', 'Standard_D2s_v3'],
-    ['PRODUCTION_READY', 'Standard_D4s_v3'],
+    [AZURE_NODE_PROFILE_NAMES.SINGLE_NODE, 'Standard_D2s_v3'],
```

Comment:
I think it might be simpler (and consistent with Christian's default objects) to make `AZURE_DEFAULT_INSTANCE TYPES` an object with fields:
`const AZURE_DEFAULT_INSTANCE_TYPES = {
    [AZURE_NODE_PROFILE_NAMES.SINGLE_NODE]: 'Standard_D2s_v3',

Reply:
Have refactored for Azure and AWS based on our last conversation. PTAL when you have a moment. Thanks!